src/Symfony/Component/Lock/LockInterface.php

```diff
@@ -43,7 +43,7 @@ public function acquire($blocking = false);
      * @throws LockConflictedException If the lock is acquired by someone else
      * @throws LockAcquiringException  If the lock can not be refreshed
      */
-    public function refresh(/* $ttl = null */);
+    public function refresh($ttl = null);
```

Comment:
that change is a BC break. We cannot uncomment it until 5.0

Reply:
Argh I didn't see that thanks